sympy/printing/tests/test_llvmjit.py

```diff
@@ -0,0 +1,45 @@
+
+from sympy.external import import_module
+
+
+if import_module('llvmlite'):
+    import sympy.printing.llvmjitcode as g
+else:
+    disabled = True
```

Comment:
I don't know - I was following the theano printer as an example, and that test includes this bit of code.   It appears to be checked in conftest.py ,in check_disabled.  I assume this is to disable the tests if the dependency is not installed (?)